modules/nw-egress-ips-about.adoc

```diff
@@ -80,6 +76,13 @@ The annotation value is an array with a single object with fields that provide t
 * `ifaddr`: Specifies the subnet mask for one or both IP address families.
 * `capacity`: Specifies the IP address capacity for the node. On AWS, the IP address capacity is provided per IP address family. On Azure and GCP, the IP address capacity includes both IPv4 and IPv6 addresses.
 
+Automatic attachment and detachment of egress IP addresses for traffic between nodes are available. This allows for traffic from many pods in namespaces to have a consistent source IP address to locations outside of the cluster. This also supports OpenShift SDN and OVN-Kubernetes, which is the default networking plug-in in Red Hat OpenShift Networking in {product-title} {product-version}.
```

Comment:
s/ in Red Hat OpenShift Networking/for Red Hat OpenShift Networking ?? There are three instances of _in_ in close company. Reads a bit weird.